pkg/fortify/reporting.go

```diff
@@ -70,7 +70,9 @@ func CreateCustomReport(data FortifyReportData, issueGroups []*models.ProjectVer
 
 		detailTable.Rows = append(detailTable.Rows, row)
 	}
+
 	scanReport.DetailTable = detailTable
+	scanReport.SuccessfulScan = data.Violations == 0
```

Comment:
Okay so just to make sure I'm reading this right- this line sets a successful scan as true if there are no data violations?

Reply:
Correct 👍